### PR TITLE
Release notes for Istio 1.14.5

### PR DESCRIPTION
**Pull request by lei-tang**
Please provide a description for what this PR is for.
Release notes for Istio 1.14.5. 

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by lei-tang** [comment]
A do-not-merge label is added to only mege the PR upon release.

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Istio 1.14.5
+linktitle: 1.14.5
+subtitle: Patch Release
+description: Istio 1.14.5 patch release.
+publishdate: 2022-10-07
+release: 1.14.5
+---
+
+This release includes security fixes in go 1.18.7 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.5/index.md:10]: Go

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+This release also includes fixes to improve robustness.
+This release note describes what’s different between Istio 1.14.4 and Istio 1.14.5.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** `kube-inject` crashes when the pod annotation `proxy.istio.io/config` is set.
+
+- **Fixed** an issue where user can not delete `iop` resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.5/index.md:20]:
```suggestion
- **Fixed** an issue where user can not delete the Istio Operator resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
```

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- **Fixed** an issue that default `idleTimeout` for passthrough cluster has been changed to `0s` since 1.14.0 and timeout is disabled. Previous behavior is using Envoy's default value.
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.5/index.md:22]: Comment added ini other RN: I'm not really sure what this fix is about. We mention pre 1.14.0 we used Envoy's default value. In 1.14.0 the default is 0 and the timeout is disabled. Now we fixed it, but what did we fix it to?

Also changes in wording.

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+linktitle: 1.14.5
+subtitle: Patch Release
+description: Istio 1.14.5 patch release.
+publishdate: 2022-10-07
+release: 1.14.5
+---
+
+This release includes security fixes in go 1.18.7 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
+This release also includes fixes to improve robustness.
+This release note describes what’s different between Istio 1.14.4 and Istio 1.14.5.
```
Comment (kfaseela) [inline content/en/news/releases/1.14.x/announcing-1.14.5/index.md:12]: what is different between

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md (hunk 3 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.5/index.md:25]: @fatedier Can you take a look at the comment above?

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md (hunk 2 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.5/index.md]: Done.

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md (hunk 1 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.5/index.md]: Done.

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md (hunk 3 above)
Comment (ericvn) [reply content/en/news/releases/1.14.x/announcing-1.14.5/index.md:25]: The 1.15.2 RN PR has a wording change and this was my latest suggestion there:

```suggestion
- **Fixed** an issue that the default `idleTimeout` for the passthrough cluster was changed to `0s` in 1.14.0, disabling the timeout. Restored the previous behavior to using Envoy's default value of 1 hour. ([Issue #41114](https://github.com/istio/istio/issues/41114))
```

Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.5/index.md]: Thanks, Eric. Your suggestion is merged.

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.14.5
+linktitle: 1.14.5
+subtitle: Patch Release
+description: Istio 1.14.5 patch release.
+publishdate: 2022-10-07
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.5/index.md:6]: Will need to update this to the correct date (10-11)?

Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.5/index.md]: Done, thanks!

File: content/en/news/releases/1.14.x/announcing-1.14.5/index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** some `ServiceEntry` host names can cause non-deterministic Envoy routes.
+  ([Issue #38678](https://github.com/istio/istio/issues/38678))
+
+- **Fixed** `kube-inject` crashes when the pod annotation `proxy.istio.io/config` is set.
+
+- **Fixed** an issue where user can not delete the Istio Operator resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.5/index.md:23]:
```suggestion
- **Fixed** an issue where the user can not delete the Istio Operator resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**